apps/dg/english.lproj/function_strings.json

```diff
@@ -234,7 +234,7 @@
       {
         "name": "date",
         "type": "string",
-        "description": "The specified date. May be a string or a computed date."
+        "description": "The specified date. May be a string or a computed date. Note that the string needs to be inside quotes."
```

Comment:
I would suggest that instead of adding this sentence, we do it as:
May be a string (in quotes) or a …